### PR TITLE
explain use case for Server Key auth

### PR DESCRIPTION
**Pull request by bradoyler**
@IonicaBizau Thought this might help first-timers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ Youtube.authenticate({
 ```
 
 #### Server Key
+_only for request that don't require [user authorization](https://developers.google.com/youtube/v3/guides/authentication) (certain list operations)_
```
Comment (IonicaBizau) [inline README.md]: Could you please use `*` instead of `_` for italic? I always used to use `*`. :smile:

**Comment by IonicaBizau** [comment]
Thanks! :sparkles: 

Related to this, maybe it's time to make the JWT issue clearer: is there any case when JWT authentication would work with this module? If not, we should remove it from docs and code.

**Comment by bradoyler** [comment]
@IonicaBizau good point. No, I don't see a use for it at all.

**Comment by bradoyler** [comment]
I'll create another PR with the 2 README changes.

**Comment by IonicaBizau** [comment]
:+1: Great. Don't forget to add yourself as contributor. :smile: 

You have to :boom: the JWT things [here](https://github.com/IonicaBizau/youtube-api/tree/master/.blah) and [here](https://github.com/IonicaBizau/youtube-api/blob/4e1167d538992e6f727b80f3c52b598ae2b53235/lib/index.js#L36-L44).

Thanks!

File: README.md (hunk 1 above)
Comment (IonicaBizau) [inline README.md]: Also, a little typo: *request* :arrow_right: *requests*, and use uppercase *O* (*Only for requests ...*). :balloon:

**Pull request closed without merging**